### PR TITLE
[add] Ability to pass an object of callbacks to ‘on’ and ‘off’

### PR DESCRIPTION
**Pull request by solomonhawk**

### CODE REVIEW

**Comment by solomonhawk** [comment]
This lib uses `zuul` for testing. I couldn't get it to run the tests properly - kept getting an error about environments. Spent 5 minutes and couldn't fix. I'd like to come back to this and re-do the entire repo from build system to implementation.

File: index.js [hunk 1]
```diff
@@ -1,13 +1,33 @@
-
 var synth = require('synthetic-dom-events');
 
-var on = function(element, name, fn, capture) {
-    return element.addEventListener(name, fn, capture || false);
-};
+function isObject(item) {
+    return (Object.prototype.toString.call(item) == '[object Object]');
```
Comment (mzlock) [inline index.js]: Maybe triple-equals that?

**Comment by mzlock** [comment]
Looks alright to me :+1:

File: index.js (hunk 1 above)
Comment (nhunzaker) [inline index.js]: The lodash version seems pretty extractable:

https://github.com/lodash/lodash/blob/9025c4a6571f65bb84abc25920b0365792ab561c/lodash.src.js#L8200-L8205

What do you think about plucking it out?

**Comment by nhunzaker** [comment]
:+1:

**Pull request merged**